 <node name="controller_spawner" pkg="controller_manager" type="spawner" output="screen" args="--namespace=/robot
        joint_state_controller
        joint1_position_controller
        joint2_position_controller
        joint3_position_controller
        --shutdown-timeout 3">
    </node>
    
  
    
        <node pkg = "assignment2"  
          type = "finite_statemachine.py"      
          name = "Finite_State_Machine" 
          launch-prefix = "xterm -fa 'Monospace' -fs 10 -e"> 
        </node>

    <node pkg = "assignment2"  
          type = "robot_states.py" 
          name = "Robot_States"
          launch-prefix = "xterm -fa 'Monospace' -fs 10 -e"> 
    </node>
    
    <node pkg = "arch_skeleton"  
          type = "planner.py"
          name = "lanner"> 
    </node>
    
    <node pkg = "arch_skeleton"  
          type = "controller.py"
          name = "controller"> 
    </node>
